Makefile

```diff
@@ -5,7 +5,10 @@ mandir ?= $(PREFIX)/share/man
 DOCKER ?= podman
 DOCS_VERSION ?= next
 RUN_TESTS ?= ci
-BACKEND ?= lorax-composer
+ifeq ($(BACKEND),)
```

Comment:
How is this different from `BACKEND ?= lorax-composer` ?

Reply:
IDK what the other syntax does, my intention here is to have the BACKEND variable default to "lorax-composer" if it is not set. It looks like that is the case without the if condition, right ?

Reply:
Right, the ?= syntax in a makefile sets it to that default if there is no environmental variable set